### ISSUE TITLE
How to place raster images in correct place?

### ISSUE THREAD

**Issue by dakom** [jkenlooper/piecemaker#15]
I don't see any documentation for the JSON files, like which to use for this or how to values translate to correct coordinates...

**Comment by dakom** [comment]
I see - it's in the generated size's `pieces.json`, where the array is interpreted as `[x,y,width,height]`

Leaving the issue open since you may want to close it when adding to documentation :)

**Comment by jkenlooper** [comment]
Yeah, haven't taken the time to document things too much.  Will document better once it reaches a more stable major version.
